ansibledriver/service/lifecycle.py

```diff
@@ -1,26 +1,35 @@
 import uuid
 import logging
 from ignition.service.framework import Capability, Service, interface
+from ignition.service.config import ConfigurationPropertiesGroup
 from ignition.service.lifecycle import LifecycleDriverCapability
 from ignition.model.lifecycle import LifecycleExecuteResponse
 
 logger = logging.getLogger(__name__)
 
+
+class AdditionalLifecycleProperties(ConfigurationPropertiesGroup, Service, Capability):
+
+    def __init__(self):
+        super().__init__('lifecycle')
+        self.keep_scripts = False
+
```

Comment:
Is it worth adding this to the default config file and Helm values file too, to show that it can be configured?